src/entities/Employer.postgres.ts

```diff
@@ -39,4 +39,8 @@ export default class Employer extends BaseEntity {
     cascade: ['remove'],
   })
   jobPosts!: JobPost[]
+
+  static localLogin(email: string, password: string) {
+    return this.find({ where: { email: email, password: password } })
+  }
```

Comment:
I don't think you're using this anywhere. You are using `findOne` already in the passport config. Can you remove it ?

Reply:
I'm using this in the employerLogin file to connect passport strategy with controller. Shall I remove the file?